packages/app/app/utils.js

```diff
@@ -16,7 +16,7 @@ export function formatDuration(duration) {
     seconds = '0' + seconds;
   }
 
-  if (hours === '00') {
+  if (String(hours) === '00') {
```

Comment:
This will need to be reviewed in details since we sometimes use numbers (if you just do a `String(0)` this will never be true)

Reply:
In the code, `hours` was checked for less than `10` so it cover the case `hours = 0` 
```js
  let hours = Math.floor(secNum / 3600);

  if (hours < 10) {
    hours = '0' + hours;
  }

  if (String(hours) === '00') {
    return minutes + ':' + seconds;
  }
```